### DISCUSSION TITLE
seam position

### DISCUSSION THREAD

**Discussion by Ahmed-mmm** [supermerill/SuperSlicer#3010] [Q&A]
please someone explain the meaning of **cost-based** mechanism 
what is angle or travel cost 
80% or whatever of what ??
this setting needs a description

**Comment by supermerill** [comment]
There are tooltip on them.
What is missing or missexplained?

**Reply by Ahmed-mmm** [reply]
The tooltip doesn't explain how do these numbers work?
how does SuperSlicer interpret these numbers to place the seam ?
are these numbers limit for movement or something else ?
could you include an example of choosing different numbers

i'm using version 2.3.57 in case the newer version has new explanation 

thanks
